private/with-npm-packages/README.md

```diff
@@ -0,0 +1,116 @@
+| 3  |    1000   |       5000       |         1        |    app    | 3405  |
+| 4  |    1000   |       5000       |         1        |    p#A    | 2091  |
+| 5  |    5000   |       1000       |         1        |    app    | 6602  |
+| 6  |    5000   |       1000       |         1        |    p#A    | 3955  |
+| 7  |    1000   |       5000       |       1000       |    app    | 3547  |
+| 8  |    1000   |       5000       |       1000       |    p#A    | 1569  |
+| 9  |    1000   |       5000       |       1000       |    p#B    | 1528  |
+| 10 |    1000   |       5000       |       5000       |    app    | 4547  |
+| 11 |    1000   |       5000       |       5000       |    p#A    | 3365  |
+| 12 |    1000   |       5000       |       5000       |    p#B    | 3127  |
+
+> Note: The times showed in the table above correspond to the times on code change with meteor cached.
+
+### Analysis
+
+After the previous results on the experiment we can clearly see that:
+
+- Changing on package is faster than doing it on app level. The reason for that is that with babel compilation we have gained paralelism in package compilation and transpilation on modules on demand. Meteor only has to put together the app modules and NPM dependencies. (See `3` and `4`, and `10`, `11` and `12`).
+
+In comparision with [the previous experiment](../with-meteor-packages/README.md#Analysis), we can clearly see that:
+
+- Meteor apps are better optimized than meteor packages, but meteor builder is a bit outdated in terms of project construction. For the moment, we can only ensure that having NPM packages and an exernal way to do ES6 transpilation for those, is better than having Meteor to handle in its own means.
+
+As always, during the experiment I also noted that:
+
+- Having large amount of files and links between them is critical on app level and makes the initial load of the builder (timings without the cache) to get so much time to rebuild (many many minutes! insane!).
+
+### Improvements
+
+Meteor builder seems to not be optimized on projects with large amount of files. For that I really think that adopting this approach should consider the next improvements:
+
+- Describe the less modules and files on app level. The application level should only have modules that depends on build-in Meteor stuff (like subscriptions, collections and etc) and the startup procedures. Maybe this can be enhanced further by describing the modules that depend on Meteor in NPM packages but as module factories, and use the app level to instantiate those modules by passing the needed Meteor context.
+
+- Adopt Webpack for package compilation is not possible apparently, as Webpack not only transpile ES6, also perform Meteor builder work, that is joining all modules together. But in our package level we want to have modules and submodules defined, not only single entry points generated by Webpack build. That is why babel transpilation is our best bet so far.
```

Comment:
I believe that is good news, it's better to use Meteor + Babel than Meteor + Webpack because Meteor is already working with Babel anyway.